lambdas/main.go

```diff
@@ -366,6 +379,8 @@ func main() {
 		lambda.Start(terminate.Handler)
 	case "transient":
 		lambda.Start(transientHandler)
+	case "management":
```

Comment:
it shouldn't be additional entry, but this function should be called by the status function when we pass `type: status`

Reply:
Added back so that it can be invoked from `status`.